.travis.yml

```diff
@@ -14,12 +14,18 @@ cache:
   directories:
     - 'node_modules'
 before_install:
+  - tmpdaemon=$(mktemp)
+  - sudo jq '."registry-mirrors" += ["https://mirror.gcr.io"]' /etc/docker/daemon.json > $tmpdaemon
+  - sudo mv $tmpdaemon /etc/docker/daemon.json
+  - sudo systemctl daemon-reload
+  - sudo service docker restart
```

Comment:
Now Docker on TravisCI is using `mirror.gcr.io`:

![swappy-20210112-145038](https://user-images.githubusercontent.com/1215541/104330387-d738cd00-54e5-11eb-8eb5-08a33e46e46a.png)

Reply:
Cool. Nice solution :+1: